src/models/forms/neuburgForm.js

```diff
@@ -1,5 +1,93 @@
 // @flow
 
 export default {
-  temp: Number
+  // landlord's data
+  surname: {
+    type: String,
+    required: true
+  },
+  firstName: {
+    type: String
+  },
+  street: {
+    type: String
+  },
+  zip: {
+    type: Number,
+    min: 10000,
+    max: 99999
+  },
+  location: {
```

Comment:
It's the landlord's place of residence (-> see wiki). But I would also suggest to pick a better name for this field, especially it could be interpreted as the location of the living offer.

Reply:
Or maybe we can structure our form in multiple objects, just like the actual form is structured: `
```
landlordInformation: {
....
},
offer: {
...
},
costs: {
...
}
```
Not sure, if that's possible with mongoose :)

Reply:
@klinzo We already have `city` in the "outer" form (the Offer), where it represents the city that is offering the living service (i.e. Neuburg).
@Schedulaar What you mean are mongoose subdocuments. I'll think about using them, but maybe it would add unnecessary logic. The names you suggested would be fine, too.